make_helpers/build_macros.mk

```diff
@@ -414,8 +421,12 @@ $(BIN): $(ELF)
 	@echo "Built $$@ successfully"
 	@${ECHO_BLANK_LINE}
 
+$(SREC): $(ELF)
+	@echo "  SREC    $$@"
+	$$(Q)$$(OC) -O srec $$< $$@
+
 .PHONY: bl$(1)
-bl$(1): $(BIN) $(DUMP)
+bl$(1): $(SREC) $(BIN) $(DUMP)
```

Comment:
We can keep the SREC rules here, but the dependency of BL1 on SREC is specific to your platform and it should be in ``platform.mk``.